code/modules/mob/grab/grab_object.dm

```diff
@@ -83,6 +87,8 @@
 	return TRUE
 
 /obj/item/grab/dropped()
+	if(ismob(loc) && !QDELETED(src))
+		to_chat(loc, SPAN_NOTICE("You release your hold on \the [affecting]."))
```

Comment:
I'd recommend putting this in `Destroy` instead. It's possible `dropped` is called multiple times, I believe.